### PR TITLE
Add Checklist Item and Notification to DB.

### PR DESCRIPTION
**Pull request by hiwaylon**
This change only supports basic inserts. Query to come, courtesy of @aaronj1335

### CODE REVIEW

**Comment by hiwaylon** [comment]
@aaronj1335 👀 @zhiqiao 👀

File: app/src/main/java/com/g11x/checklistapp/data/Database.java [hunk 1]
```diff
@@ -70,12 +72,94 @@ private static String createCreateTableSql() {
...
+  }
+
   private static final UriMatcher URI_MATCHER = createContentRouter();
 
   private static UriMatcher createContentRouter() {
     UriMatcher matcher = new UriMatcher(UriMatcher.NO_MATCH);
 
     matcher.addURI("com.g11x.checklistapp.provider", Database.NAME + "/" + ImportantInformation.TABLE_NAME, 1);
+    matcher.addURI("com.g11x.checklistapp.provider", Database.NAME + "/" + ChecklistItem.TABLE_NAME, 2);
+    matcher.addURI("com.g11x.checklistapp.provider", Database.NAME + "/" + Notification.TABLE_NAME, 2);
```
Comment (aaronj1335) [inline app/src/main/java/com/g11x/checklistapp/data/Database.java]: I think this should be 3. Also why just a number? Constants seem better.

File: app/src/main/java/com/g11x/checklistapp/data/Database.java [hunk 2]
```diff
@@ -70,12 +72,94 @@ private static String createCreateTableSql() {
...
+    }
+  }
+
   private static final UriMatcher URI_MATCHER = createContentRouter();
 
   private static UriMatcher createContentRouter() {
     UriMatcher matcher = new UriMatcher(UriMatcher.NO_MATCH);
 
     matcher.addURI("com.g11x.checklistapp.provider", Database.NAME + "/" + ImportantInformation.TABLE_NAME, 1);
+    matcher.addURI("com.g11x.checklistapp.provider", Database.NAME + "/" + ChecklistItem.TABLE_NAME, 2);
```
Comment (aaronj1335) [inline app/src/main/java/com/g11x/checklistapp/data/Database.java]: how about `Constants.LOCAL_REPOSITORY_AUTHORITY`

**Comment by aaronj1335** [comment]
only one major thing. also i get the email notifications, so no need to side-eye-emoji me (unless you prefer, whatever).

File: app/src/main/java/com/g11x/checklistapp/data/Database.java (hunk 1 above)
Comment (hiwaylon) [inline app/src/main/java/com/g11x/checklistapp/data/Database.java]: Done. Done.

File: app/src/main/java/com/g11x/checklistapp/data/Database.java (hunk 2 above)
Comment (hiwaylon) [inline app/src/main/java/com/g11x/checklistapp/data/Database.java]: Done.

**Comment by aaronj1335** [comment]
:shipit:

**Pull request merged**